.gitattributes

```diff
@@ -0,0 +1,3 @@
+rzk/grammar/Syntax.cf linguist-language=grammar
```

Comment:
I don't see `grammar` as a language identifier in [`languages.yml`](https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml). Perhaps it should be `ABNF` or `EBNF`? Note that either would also require `linguist-detectable` since they have `type: data`

Reply:
Updated to use `EBNF`, thanks!